central/complianceoperator/v2/pipelines/complianceoperatorinfo/pipeline.go

```diff
@@ -72,7 +74,15 @@ func (s *pipelineImpl) Run(ctx context.Context, clusterID string, msg *central.M
 
 	// if not ready, add it to the status errors
 	if readyPods < desiredPods {
-		operatorErrors = append(operatorErrors, fmt.Sprintf("compliance operator not ready.  Only %d pods are ready when %d are desired.", readyPods, desiredPods))
+		operatorErrors = append(operatorErrors, fmt.Sprintf("Compliance operator not ready. Only %d out of desired %d are ready.", readyPods, desiredPods))
+	}
+
+	// we support only newer versions of compliance operator
+	complianceOperatorVersion, err := semver.NewVersion(operatorInfo.GetVersion())
+	if complianceOperatorVersion == nil || err != nil {
```

Comment:
should we log the error?

Reply:
I also think that it maybe helpful to wrap and log the error from semver

Reply:
@janisz resolving as the log was added. Feel free to unresolve if you have further comments.